pages/common/rails-db.md

```diff
@@ -0,0 +1,35 @@
+# rails db
+
+> Rails database actions.
+
+- Access database console:
+
+`rails db`
+
+- Create databases defined in current environment:
+
+`rails db:create`
+
+- Destroy databases defined in current environment:
+
+`rails db:drop`
+
+- Run pending migrations:
+
+`rails db:migrate`
+
+- View status of each migration file:
+
+`rails db:migrate:status`
+
+- Rollback the last migration:
+
+`rails db:rollback`
+
+- Fill database with data defined in db/seeds.rb:
+
+`rails db:seed`
+
+- Create databases, load the schema, and initialize with seed data:
```

Comment:
Generally we like tldr-pages to have the examples in the order you're  likely to use them. This command looks like it might be useful first - so maybe it should be at the top?

Reply:
Makes sense to me, I'm okay with moving this to the top.